Event/OAuthEvent.php

```diff
@@ -37,13 +37,19 @@ class OAuthEvent extends Event
     private $isAuthorizedClient;
 
     /**
+     * @var string
+     */
+    private $scopes;
+
+    /**
      * @param UserInterface $user
      */
-    public function __construct(UserInterface $user, ClientInterface $client, $isAuthorizedClient = false)
+    public function __construct(UserInterface $user, ClientInterface $client, $isAuthorizedClient = false, $scopes = '')
```

Comment:
With the call at line L96, scopes could be null or a string.
So `$scopes = ''` should be `$scopes = null` also. Or you can change the default value at line L96.